jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -1,7 +1,9 @@
+
 #!/bin/bash
 
 source "${LAUNCH_DIR}/launch-common.sh"
 source "${LAUNCH_DIR}/logging.sh"
+source "${JBOSS_HOME}/bin/launch/jboss-kie-common.sh"
```

Comment:
I see that you are sourcing the new jboss-kie-common.sh script inside jboss-kie-smartrouter.sh. However, you also need to remove the extracted code out of jboss-kie-wildfly-common.sh, and source it there as well. It should be part of this PR.